ruby_review_week_four.rb

```diff
@@ -1,37 +1,95 @@
+def findstring(array,string)
+  indexstring = array.index(string)
+  if indexstring != nil
+    return true
+  else
+    return false
+end
+end
 
-# 1. What is a variable?
 
-# 2. What Ruby method can you use to make a loop from an array?
-
-# 3. In Ruby, what are booleans?
-
-# 4. How do classes relate to objects?
-
-# 5. What is an instance variable?
-
-# 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
-
-# 7. What is handy about using gems in Ruby?
-
-# 8. Define a class called fish, and leave it empty.
-
-# 9. How would you use that class to make three fish objects?
-
-# 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
-
-# 11. 
-
-# a. Create one fish object and store it in a variable, and 
-
-# b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
-
-# 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
-
-# 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
-
-# 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
 
 # 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise. 
-
-# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
\ No newline at end of file
+def divisibility3(number)
+  number % 3 == 0
+end
+    
+# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
+def remainders(array)
+  arr1 = []
+  arr2 = []
+  all_numbers = [arr1,arr2]
+  array.each do |number|
+    if divisibility3(number) == true
+      arr1 << number
+    else
+      arr2 << number
+    end
+  end
+  return all_numbers
```

Comment:
nice! looks like this works.... one line of code to refactor might be 87, do we need the `== true`?